s/elements/stickpad/style.css.ts

```diff
@@ -0,0 +1,26 @@
+
+import {css} from "@chasemoskal/magical"
+
+export default css`
+	:host {
+		display: flex;
+		width: 30em;
+		height: 15em;
+	}
+
+	nub-stick {
+		width: 100%;
+		height: 100%;
+	}
+
+	nub-stick::part(base) {
+		visibility: hidden;
+		width: 50px;
+		height: 50px;
+	}
+
+	nub-stick[data-visible]::part(base) {
+		visibility: visible;
+		position: absolute;
+	}
```

Comment:
i think we should make the nub-stick visible by default, in the center of the pad rectangle -- basically, for new players, if they don't initially see the stick, they might not realize it's available at all -- let's make sure this is customizable by the developer using shadow parts.

i also noticed the stickpad stick is smaller than the ordinary stick -- we should make them both the same size by default.